services/asset/nodes/schemas.go

```diff
@@ -124,30 +136,72 @@ func makeNodeSchema(recursiveNodes map[string]*schema.Schema) map[string]*schema
 }
 
 var dataSourceFilterSchema = map[string]*schema.Schema{
-	"parent_node_ids": {
+	"created_by": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the user who created the Node",
+	},
+	"from_created_at": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the Node creation date, using ISO-8601 format. Properties with a creation date greater or equals than the filter value will be selected (if they are not excluded by other filters)",
+	},
+	"from_last_modified_at": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the Node last modification date, using ISO-8601 format. Nodes with a last modification date greater or equals than the filter value will be selected (if they are not excluded by other filters)",
+	},
+	"last_modified_by": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the user who modified last the Node",
+	},
+	"to_created_at": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the Node creation date, using ISO-8601 format. Nodes with a creation date lower or equals than the filter value will be selected (if they are not excluded by other filters)",
+	},
+	"to_last_modified_at": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the Node last modification date, using ISO-8601 format. Nodes with a last modification date lower or equals than the filter value will be selected (if they are not excluded by other filters)",
+	},
+	"ids": {
 		Type:     schema.TypeList,
 		Optional: true,
 		Elem: &schema.Schema{
 			Type:         schema.TypeString,
 			ValidateFunc: validation.IsUUID,
 		},
+		Description: "Only returns node who's id matches one of the provided values.",
```

Comment:
```suggestion
		Description: "Only returns node whose id matches one of the provided values.",
```
Not sure about this one

Reply:
thanks :)